opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -503,12 +462,28 @@ def train_model(
             model, optimizer, train_dataloader, scheduler = accelerator.prepare(
                 model, optimizer, train_dataloader, scheduler
             )
+            print("Device using for training: ", accelerator.device)
             model.to(accelerator.device)
             init_time = time.time()
             total_loss = []
 
+            if accelerator.process_index == 0:
+                print("Start training with accelerator...\n")
+
+                print(f"The number of training epochs per process are {num_epochs}\n")
+
+                print(
+                    f"The total number of steps per training epoch are {len(train_dataloader)}\n"
+                )
+
```

Comment:
Add comment about training loop here 

"The following training loop trains the sentence transformer model using the standard contrastive loss with in-batch negatives. The particular contrastive loss that we use is the Multi-class N-pair Loss (eq. 6, Sohn, NeurIPS 2016). In addition, we symmetrize the loss with respect to queries and passages (as also used in OpenAI's CLIP model). The performance improves with the number of in-batch negatives but larger batch sizes can lead to out of memory issues, so please use the batch-size judiciously. "